packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -1,31 +1,88 @@
-                {/* [Todo] needs to update tabs component children instead of using label property */}
-                <div label={localize('Dashboard')}>
-                    <div>
-                        <h1>Create or start a bot</h1>
-                    </div>
-                </div>
-                <div label='Quick Strategy'>
-                    <div>Contennt 2</div>
-                </div>
-                <div label='Bot Builder'>
-                    <div>Contennt 3</div>
-                </div>
-                <div label='Tutorial'>
-                    <div>Contennt 4</div>
-                </div>
-            </Tabs>
+        <div className='dashboard__main'>
+            <div className={classNames('dashboard__container', { 'w-100': !show_side_bar })}>
+                <ReactJoyride steps={DBOT_ONBOARDING} run={tour_run} continuous={true} showProgress={true} />
+                <Tabs active_index={active_tab} onTabItemClick={setActiveTab} top>
+                    {/* [Todo] needs to update tabs comIcDashBoardComponentsTabponent children instead of using label property */}
+                    <Tab icon='IcDashboardComponentTab' label={localize('Dashboard')}>
+                        <DashboardComponents />
+                    </Tab>
+                    <Tab icon='IcBotbuilderTabIcon' label={localize('Bot Builder')} id='id-bot-builder'>
+                        <BotBuilder />
+                    </Tab>
+                    <Tab
+                        icon='IcQuickStrategyIcon'
+                        label={localize('Quick Strategy')}
+                        id='id-quick-strategy'
+                        onTabItemClick={handleClick}
+                    >
+                        <div
+                            className={classNames('quick-strategy', {
+                                'quick-strategy__notifications-container--open': !is_drawer_open,
+                            })}
+                        >
+                            <QuickStrategy />
+                        </div>
+                    </Tab>
+                    <Tab icon='IcChartsTabDbot' label={localize('Charts')} id='id-charts'>
+                        <div className='dashboard__chart-wrapper'>
+                            <Chart />
+                        </div>
+                    </Tab>
+                    <Tab icon='IcTutorialsTabs' label={localize('Tutorial')} id='id-tutorials'>
+                        <div>{localize('Under Developments')}</div>
+                    </Tab>
+                </Tabs>
+            </div>
+
+            <div
+                className={classNames('dashboard__run-strategy-wrapper', {
+                    'dashboard__sidebar-wrapper--active': active_tab !== 0 || !show_side_bar,
+                })}
+            >
+                <RunStrategy />
+                {active_tab === 0 && <Sidebar is_sidebar_open={show_side_bar} setSideBarState={setShowSideBar} />}
```

Comment:
Let's move this condition after `{(active_tab !== 0 || !show_side_bar) && <RunPanel />}` or change this condition to `active_tab === 0 && show_side_bar` unless for `active_tab === 0` and `!show_side_bar` it won’t happen